### PR TITLE
Extentsion to logs

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR addresses the issue where a 404 "Page Not Found" error occurs when clicking on "View" or "Download" in the event metadata, specifically when not on the root page.

Additionally, this PR includes the addition of a file extension to the logs file.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/registry/generate.go [hunk 1]
```diff
@@ -79,14 +79,14 @@ mesheryctl registry generate --registrant-def [path to connection definition] --
 			return ErrUpdateRegistry(err, modelLocation)
 		}
 		utils.Log.SetLevel(logrus.DebugLevel)
-		logFilePath := filepath.Join(logDirPath, "Logs")
+		logFilePath := filepath.Join(logDirPath, "Logs.txt")
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/registry/generate.go:82]:
```suggestion
		logFilePath := filepath.Join(logDirPath, "model-generation.log")
```

Are these model-generation or registry-import logs? Either way, let's name accordingly. Please

Comment (leecalcote) [reply mesheryctl/internal/cli/root/registry/generate.go:82]: It's it's both, then `registry.log` will work.

File: mesheryctl/internal/cli/root/registry/generate.go [hunk 2]
```diff
@@ -79,14 +79,14 @@ mesheryctl registry generate --registrant-def [path to connection definition] --
...
 		logFile, err = os.Create(logFilePath)
 		if err != nil {
 			return err
 		}
 
 		utils.LogError.SetLevel(logrus.ErrorLevel)
-		logErrorFilePath := filepath.Join(logDirPath, "Errors")
+		logErrorFilePath := filepath.Join(logDirPath, "Errors.txt")
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/registry/generate.go:89]:
```suggestion
		logErrorFilePath := filepath.Join(logDirPath, "registry-errors.log")
```

File: mesheryctl/internal/cli/root/registry/generate.go (hunk 1 above)
Comment (Jougan-0) [reply mesheryctl/internal/cli/root/registry/generate.go:82]: These are the logs of `Model component Generator`. In the action we only want to mail the `error` so it's better we have 2 separate logs one that shows the error and the other that stores the general logs.

Comment (leecalcote) [reply mesheryctl/internal/cli/root/registry/generate.go]: Will you add a note here highlighting how thoughtful we have been and how users can take note that a missing `model-generation-errors.log` is a good thing, but that it and the `model-generation.log` files can be referenced? Explain how they are used, please.

**Pull request merged**